Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
 members = [
-    "rs"
+    "rs",
+    "rs-tonic",
```

Comment:
fwiw, I'm in favor of just replacing the `rs` crate

Reply:
yup, will do